src/LRS.js

```diff
@@ -310,7 +343,11 @@ TinCan client library
         saveStatement: function (stmt, cfg) {
             this.log("saveStatement");
             var requestCfg,
-                versionedStatement;
+                versionedStatement,
+                boundary,
+                flag = true,
+                attachmentContent = [],
+                a;
```

Comment:
Getting *really* nitpicky now :-), but `i` is more canonical in the lib (AFAICT all for loops use it), so let's switch for consistency.